docs/docs/starters.md

```diff
@@ -2,45 +2,65 @@
 title: "Gatsby Starters"
 ---
 
-The Gatsby CLI tool lets you install “starters”. These are boilerplate Gatsby sites maintained by the community.
+The Gatsby CLI tool lets you install **starters**, which are boilerplate Gatsby sites maintained by the community and intended for jump-starting development quickly.
 
-`gatsby new` helps you start your project by cloning the boilerplate, installing dependencies, and clearing Git history.
+## Installing starters
 
-When creating a new site, you can optionally specify a starter to base your new site on, either with the `[URL]` of the `[GIT_USER_NAME/REPO]` e.g.
+Execute the `gatsby new` command to clone a boilerplate starter, install its dependencies, and clear Git history.
 
-`gatsby new [SITE_DIRECTORY] [URL_OF_STARTER_GITHUB_REPO]`
-or
-`gatsby new [SITE_DIRECTORY] [GIT_USER_NAME/REPO]`
+### Using Git repo URLs
```

Comment:
Added some headers here and grouped the commands by technique, so hopefully that makes it more digestible! I also edited the intro to limit confusion with regards to git history.